compute/admin_guide/agentless-scanning/onboard-accounts/configure-gcp.adoc

```diff
@@ -0,0 +1,93 @@
+:toc: macro
+[#configure-gcp-agentless]
+== Configure Agentless Scanning for GCP
+
+toc::[]
+
+=== Prerequisites
+
+You need the following items to onboard a GCP account for agentless scanning.
+
+* A GCP project with the permissions needed to create a service account and roles under this project.
+
+* At least one Google Compute VM instance deployed with running containers to validate scanning.
+
+* A https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/get-started-with-prisma-cloud/enable-access-prisma-cloud-console[working connection] from your cloud account to Prisma Cloud.
+
+* You have https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/connect-your-cloud-platform-to-prisma-cloud/onboard-gcp[onboarded at least one GCP account into Prisma Cloud].
+
+* If you want to use a hub account to scan multiple target accounts, all the target accounts and the hub accounts must be onboarded before you configure agentless scanning.
+
+* To use encrypted volumes, grant the `Encrypter` and `Decrypter` role to the Compute Engine Service Agent to access the https://cloud.google.com/compute/docs/disks/customer-managed-encryption[Cloud KMS keys] needed.
+
+[.task]
+=== Configure Agentless Scanning for GCP Accounts
+
+[.procedure]
+
+. Log in to the Prisma Cloud administrative console.
+
+. Select *Compute > Manage > Cloud Accounts*.
+
+. Click the edit button of your cloud account.
+
+. Go to the *Agentless Scanning* section.
+
+. Expand the *Advanced settings* and provide the following information.
++
+image::agentless-gcp-pcee-advanced-settings.png[width=540]
+
+.. Enable Permissions check to verify that the permissions are correct before running a
+scan.
+.. *Scanning type*: For GCP accounts, you can decide between xref:../agentless-scanning.adoc#scanning-modes[two scanning modes].
+
+... *Same Account*: Scan hosts of your GCP account using that same account. Use this value for the account you want to use as the hub account.
+
+... *Hub Account*: Scan hosts of your GCP account using a different account. Select another onboarded account from the list to scan the account you are configuring.
+
+.. Enter a *Proxy* value if traffic leaving your GCP tenant uses a proxy.
+
+.. Under *Scan scope* you can refine the scope of the scanning by *Regions* or using tags.
++
+image::tags-scope.png[width=300]
+
+... Choose *All regions* to scan in all GCP regions.
+If you choose *Custom regions*, enter the GCP region in which you want Prisma Cloud to scan.
+
+... Choose whether to *Exclude hosts by tags* or to *Include hosts by tags*.
+You can use wildcards to refine the scanning as needed.
```

Comment:
Pleasse use this text:

You can use wildcards to specify a range of tags. Wildcards can be used in both keys and values in the following ways: `"abcd*", "*abcd", "abcd", "*", "*abcd*"`